_data/projects.json

```diff
@@ -0,0 +1,170 @@
+            {
+                "name": "OCTO Innovation Program", 
+                "email": "open@dc.gov"
+            }
+        ], 
+        "github": {
+            "languages": {
+                "Shell": 333, 
+                "HTML": 4713, 
+                "Ruby": 3166, 
+                "JavaScript": 1404, 
+                "CSS": 12367
+            }, 
+            "teams": [
+                "DC Digital Service"
+            ], 
+            "activity": []
+        }, 
+        "name": "guides-template", 
+        "license": "https://raw.githubusercontent.com/DCgov/guides-template/gh-pages/LICENSE.md", 
+        "homepage": "http://open.dc.gov/guides-template/", 
+        "links": [], 
+        "data": [], 
+        "repository": "https://github.com/DCgov/guides-template", 
+        "contact": {
+            "name": "Emanuel Feld", 
+            "email": "open@dc.gov"
+        }, 
+        "keywords": [
+            "guide", 
+            "instructions"
+        ], 
+        "geography": [], 
+        "type": "web app", 
+        "id": "https://raw.githubusercontent.com/DCgov/civic.json/master/schema.json", 
+        "description": "A template and set of instructions for setting up DCgov guide sites."
+    }, 
+    {
+        "status": "Beta", 
+        "partners": [
+            {
+                "name": "OCTO Innovation Program", 
+                "email": "open@dc.gov"
+            }
+        ], 
+        "github": {
+            "languages": {
+                "Shell": 83, 
+                "HTML": 4742, 
+                "Ruby": 3166, 
+                "JavaScript": 1404, 
+                "CSS": 12124
+            }, 
+            "teams": [
+                "DC Digital Service"
+            ], 
+            "activity": []
+        }, 
+        "name": "open source guidelines", 
+        "license": "CC0-1.0", 
+        "homepage": "http://open.dc.gov/open-source-guidelines/", 
+        "links": [], 
+        "data": [], 
+        "repository": "https://github.com/dcgov/open-source-guidelines", 
+        "contact": {
+            "name": "Emanuel Feld", 
+            "email": "opensource@dc.gov"
+        }, 
+        "keywords": [
+            "open source", 
+            "guide"
+        ], 
+        "geography": [
+            "DC"
+        ], 
+        "type": "guide", 
+        "id": "https://raw.githubusercontent.com/DCgov/civic.json/master/schema.json", 
+        "description": "A how-to guide for agencies that have joined or would like to join DC Government's Open Source Pilot program."
+    }, 
+    {
+        "status": "Beta", 
+        "partners": [
+            {
+                "name": "OCTO Innovation Program", 
+                "email": "open@dc.gov"
+            }
+        ], 
+        "github": {
+            "languages": {
+                "Python": 26641, 
+                "Shell": 1745, 
+                "HTML": 5472
+            }, 
+            "teams": [
+                "DC Digital Service"
```

Comment:
@benjaminrosenbaum that's just taken via the github API. In any case, I'm not yet displaying any information that came from the API.